.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: Tests
+permissions:
+  contents: read
+on:
+  pull_request:
+    paths-ignore:
+      - "*.md"
+      - "assets/**"
+  push:
+    paths-ignore:
+      - "*.md"
+      - "assets/**"
+    branches:
+      - main
+  workflow_dispatch:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  ubuntu:
+    name: Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install tmux
+        run: sudo apt-get update && sudo apt-get install -y tmux
+      - uses: actions/checkout@v4
+      - name: config & plugin
+        shell: bash
+        run: |
+          bash --version
+          tmux -V
+          ./.github/scripts/test.bash ./catppuccin.tmux
+
+  old-bash:
+    name: "Old bash"
+    runs-on: ubuntu-latest
+    container:
+      image: bash:3.2.57
+    steps:
+      - uses: actions/checkout@v4
+      - name: config & plugin (old bash)
+        shell: bash
+        run: |
+          # No tmux available in this container
```

Comment:
I might be misunderstanding the point of this but if tmux isn't available in this container, why would we be running the test script?

Reply:
To check if this bash version does not error on "new" syntax `catppuccin.tmux`

![image](https://github.com/catppuccin/tmux/assets/44153531/28f6fdbe-88be-4e7d-93d5-0e2a12830a1c)

commit: 11a92b95fdf9c47035e8d1c58db6e92a6a5e5b4f